src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/config/LocalStackContainer.kt

```diff
@@ -34,7 +34,7 @@ object LocalStackContainer {
       withServices(LocalStackContainer.Service.SQS, LocalStackContainer.Service.SNS)
       withEnv("DEFAULT_REGION", "eu-west-2")
       waitingFor(
-        Wait.forLogMessage(".*Running on.*", 1),
+        Wait.forLogMessage(".*Ready.*", 1),
```

Comment:
Integration tests were not running in IntelliJ for some reason - drilled down to this wait and checked other repos using LocalStackContainer and they were waiting for the container to be ready rather than running on a particular port. Changed and now run as expected.